testing/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.18.10
```

Comment:
We get rid of `ginkgo` so I guess it is no longer needed

Reply:
Devcontainer is needed to isolate elastic-agent. It turns out we already have elastic-agent installed on our machines. Also it helps us test it with unix-like different OS. That's the purpose of it. So it doesn't depend on our tooling.

Reply:
Running the agent is a container isn't the same code path as running it on your host. There are several differences in how the agent is installed.

Most (all?) of the agent engineers have opted out of have agent pre-installed on our own machines so we can install our own custom builds, or they develop for the agent in VMs.

I'd prefer to remove this if we don't absolutely need it.